.github/workflows/helm-lint-and-test.yaml

```diff
@@ -0,0 +1,36 @@
+name: helm
+
+on:
+  pull_request:
+    paths:
+      - "charts/perses/**"
+    branches:
+      - main
+
+jobs:
+  helm-lint-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: setup helm lint requirements
+        uses: ./.github/actions/setup_helmlint
+        with:
+          create-kind-cluster: "true"
+
+      - name: setup go lang
+        uses: ./.github/actions/setup_environment
+        with:
+          enable_go: true
+
+      - name: run chart-testing (install)
+        run: ct install --charts charts/perses
+
+      - name: install mdox
+        run: go install github.com/bwplotka/mdox@latest
+
+      - name: run chart README lint
+        run: make update-helm-readme && make fmt-docs && make checkdocs
```

Comment:
since `make update-helm-readme` is already running `fmt-docs` no need to do it again ;)